### PR TITLE
new token notifications

### PR DESCRIPTION
**Pull request by benny-conn**
Changes:

- **Added** notification and event types for receiving new tokens
- **Added** resolvers for the new notification type
- **Added** event dispatching and notification dispatching to token processing service

### CODE REVIEW

File: event/event.go [hunk 1]
```diff
@@ -58,12 +59,67 @@ func AddTo(ctx *gin.Context, disableDataloaderCaching bool, notif *notifications
...
+func PushEvent(ctx context.Context, evt db.Event) {
+	if hub := sentryutil.SentryHubFromContext(ctx); hub != nil {
+		sentryutil.SetEventContext(hub.Scope(), persist.NullStrToDBID(evt.ActorID), evt.SubjectID, evt.Action)
+	}
+	if err := DispatchDelayed(ctx, evt); err != nil {
+		logger.For(ctx).Error(err)
+		sentryutil.ReportError(ctx, err)
+	}
+}
+
```
Comment (benny-conn) [inline event/event.go:122]: Moved this logic here for sharing across packages

File: graphql/schema/schema.graphql [hunk 2]
```diff
@@ -1881,6 +1881,17 @@ type SomeoneViewedYourGalleryNotification implements Notification & Node & Group
...
+type NewTokensNotification implements Notification & GroupedNotification & Node @goEmbedHelper {
+  id: ID!
+  dbid: DBID!
+  seen: Boolean
+  creationTime: Time
+  updatedTime: Time
+  count: Int
+
+  token: Token @goField(forceResolver: true)
+}
```
Comment (benny-conn) [inline graphql/schema/schema.graphql:1893]: `count` would be how many they minted for example

**Review by jarrel-b — APPROVED** [review]
Looks awesome!

File: db/queries/core/query.sql [hunk 3]
```diff
@@ -290,7 +290,7 @@ select t.* from tokens t
 INSERT INTO events (id, actor_id, action, resource_type_id, user_id, subject_id, data, group_id, caption) VALUES ($1, $2, $3, $4, $5, $5, $6, $7, $8) RETURNING *;
 
 -- name: CreateTokenEvent :one
-INSERT INTO events (id, actor_id, action, resource_type_id, token_id, subject_id, data, group_id, caption, gallery_id, collection_id) VALUES ($1, $2, $3, $4, $5, $5, $6, $7, $8, $9, $10) RETURNING *;
+INSERT INTO events (id, actor_id, action, resource_type_id, token_id, subject_id, data, group_id, caption, gallery_id, collection_id) VALUES ($1, $2, $3, $4, $5, $5, $6, $7, $8, sqlc.narg('gallery'), sqlc.narg('collection')) RETURNING *;
```
[the remark this answers is not in the document]
Comment (jarrel-b) [reply db/queries/core/query.sql:293]: Ah makes sense!

**Pull request merged**